src/pages/storage/forms/StoragePoolFormMain.tsx

```diff
@@ -68,6 +68,13 @@ const StoragePoolFormMain: FC<Props> = ({ formik }) => {
     ? "Cannot rename storage pools"
     : undefined;
 
+  const cephObjectNotice = (
+    <>
+      Rados gateway must be enabled for CephObject driver to work. If using
```

Comment:
```suggestion
      Rados gateway must be enabled for Ceph Object driver to work. If using
```
Let's use the same formatting as in the dropdown.